WalletWasabi/Wallets/SmartBlockProvider.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.Extensions.Caching.Memory;
+using NBitcoin;
+using Nito.AsyncEx;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Wallets
+{
+	/// <summary>
+	/// SmartP2pBlockProvider is a blocks provider that can provides 
+	/// blocks from multiple requesters.
```

Comment:
I think you mean `for` not `from`.
```suggestion
	/// blocks for multiple requesters.
```

Reply:
Ummm i am not sure. I think it is `from`.